lib/install.js

```diff
@@ -32,6 +37,15 @@ function install(opts, cb) {
                 return cb1();
             });
 
+            if (process.env.HOT_RELOAD) {
```

Comment:
@DonutEspresso shouldn't this be an option and passing this ENV (property) in NQ?

Reply:
Yes +1 - let's make this a first class option and we can thread through the env var from a lower level.